src/components/InfoBanner/blocks/InfoBannerCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { BoxProps, Box } from '../../Box/Box';
+import { Link as WaveLink } from '../../Link/Link';
+import { Text } from '../../Text/Text';
+
+import { ICON_VARIANTS, IconBox, InfoBannerVariants, ROLE_VARIANTS, RoundedBox } from '../InfoBanner';
+
+interface CardProps extends BoxProps {
+    children: ReactNode;
+    variant?: InfoBannerVariants;
+    emphasized?: boolean;
+}
+
+const LinkList = styled.ul`
+    margin-block-start: 0.3rem;
+    margin-block-end: 0.3rem;
+    padding-inline-start: 1.5rem;
+
+    li {
+        line-height: 1.1;
+    }
+`;
+
+const Title = ({ children }: { children: string }) => (
+    <Text className="Infobanner__Title" fontWeight="bold" textAlign="left">
+        {children}
+    </Text>
+);
+
+const Description = ({ children }: { children: string }) => (
+    <Text className="Infobanner__Description" fontSize="small" textAlign="left">
+        {children}
+    </Text>
+);
+
+const Link = ({ linkUrl, linkText }: { linkUrl: string; linkText: string }) => (
```

Comment:
To me it seem's like this link is coupled to a very specific behaviour. For example, I'm not able to pass an `onClick` prop. But maybe that's fine given it's a compound component. What do you think?

Reply:
I will close this now to move the MR, but if we feel need to add an `onClick` on `<Link>` component, we can create an issue for it then and update it here :)

Reply:
I'm definitely late to the conversation but that might be a problem if we wanted to add some tracking for example.
We could have exposed the `StyledLink` itself instead of wrapping it, because intrinsically it accepts a `children` a
`href` and also all the props from `WaveLink` 

```tsx
InfoBannerCard.Link = StyledLink;
```

Sorry for being late 😅